### PR TITLE
Add initial skeleton files

### PR DESCRIPTION
**Pull request by fjvallarino**
- The main base configuration, package, and source files.
- Supporting files for building the component
- Initial README

More skeleton project files (test infrastructure, demo, etc) to come in followup PRs

@santiagonoguez @stulees please review

### CODE REVIEW

File: src/rise-data-twitter.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console, no-unused-vars */
+
+import { RiseElement } from "rise-common-component/src/rise-element.js";
+import { timeOut } from "@polymer/polymer/lib/utils/async.js";
```
Comment (stulees) [inline src/rise-data-twitter.js]: @fjvallarino You can remove the `async`, `debounce`, and `moment` imports

Comment (fjvallarino) [reply src/rise-data-twitter.js]: @stulees updated!

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "author": "Rise Vision",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/Rise-Vision/rise-data-twitter/issues"
+  },
+  "homepage": "https://github.com/Rise-Vision/rise-data-twitter/#readme",
+  "dependencies": {
+    "@polymer/polymer": "3.1.0",
+    "@webcomponents/webcomponentsjs": "^2.1.3",
+    "rise-common-component": "git://github.com/Rise-Vision/rise-common-component.git#v1.2.1"
```
Comment (santiagonoguez) [inline package.json]: @fjvallarino  we could depend on 1.3.0 now, right ?

File: src/rise-data-twitter.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  }
+
+  // Event name constants
+  static get EVENT_DATA_UPDATE() {
+    return "data-update";
+  }
+
+  static get EVENT_DATA_ERROR() {
+    return "data-error";
+  }
```
Comment (santiagonoguez) [inline src/rise-data-twitter.js]: @fjvallarino if other components also use these constants, shouldn't we moving them up to RiseElement instead ? thanks !

Comment (fjvallarino) [reply src/rise-data-twitter.js]: @santiagonoguez I'll propose that as an improvement!

Comment (fjvallarino) [reply src/rise-data-twitter.js]: @santiagonoguez you already did 😭

**Pull request merged**